operators/olm-restricted-networks.adoc

```diff
@@ -5,22 +5,35 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-When {product-title} is installed on restricted networks, Operator Lifecycle
-Manager (OLM) can no longer use the default OperatorHub sources because they
-require full Internet connectivity. Cluster administrators can disable those
-default sources and create local mirrors so that OLM can install and manage
-Operators from the local sources instead.
+When {product-title} is installed on restricted networks, also known as a
+disconnected cluster, Operator Lifecycle Manager (OLM) can no longer use the
+default OperatorHub sources because they require full Internet connectivity.
+Cluster administrators can disable those default sources and create local
+mirrors so that OLM can install and manage Operators from the local sources
+instead.
 
 [IMPORTANT]
 ====
 While OLM can manage Operators from local sources, the ability for a given
 Operator to run successfully in a restricted network still depends on the
-Operator itself. See the following Red Hat Knowledgebase Article for details on
-Operators that support running in disconnected mode:
+Operator itself. The Operator must:
+
+* List any related images, or other container images that the Operator might
+require to perform their functions, in the `relatedImages` parameter of its
+ClusterServiceVersion (CSV) object. (For {product-title} 4.3 and later usage)
```

Comment:
Just a suggestion: for (For {product-title} 4.3 and later usage) and (For {product-title} 4.2 and later usage), I'd be inclined to remove the parentheses and make them sentences.